sof-host-defconfig

```diff
@@ -0,0 +1,10 @@
+CONFIG_TASKSTATS=n
+CONFIG_TASK_DELAY_ACCT=n
+CONFIG_KVM=m
+CONFIG_KVM_INTEL=m
+CONFIG_VHOST_NET=m
+CONFIG_VHOST_SOF=y
+CONFIG_BRIDGE=m
+CONFIG_TUN=m
+CONFIG_SND_SOC_INTEL_SOF_PCM512x_MACH=m
+CONFIG_SND_SOC_SOF_DEBUG_FORCE_IPC_POSITION=y
```

Comment:
same, why is this needed for the host? That would only deal with the DSP communication, so that should be defined in another config